### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by monheimx9** [1-rafael-1/pi-pico-alarmclock-rust#60] [General]
Just wanted to say thank you
By far this is the best resource to my learning journey about embedded rust/async with embassy
I come by every day to read it
Awesome code documentation, very clear to a noob like me
All the best and cheers ദ്ദി(ᵔᗜᵔ)

**Comment by 1-rafael-1** [comment]
@monheimx9 thanks a lot for the kind words!
I consider myself a noob, too. I had a lot of help making this. 
All the documentation I write basically to explain stuff to myself, just the way I learn. 🙂 

If you like this, you could also look at my https://github.com/1-rafael-1/simple-robot 

Also, make sure to get on the Embassy and Rust Embedded matrix channels, if you aren't already. Helped me a ton.
